trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative './base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name[0, 10]
+  end
```

Comment:
- [ ] I think here you forget to add a method to check the name If it's longer it should trim the word. unless it should just keep it as required in the requiremnets so I would recommend to update your method according to it 💯🔥🙏🏻

### I think the trim it should be somthing like this, Try this ⬇🚀💪🏻

>  @nameable.correct_name[0..9] instead of [0. 10] 

![image](https://user-images.githubusercontent.com/58553711/217405862-5e57637e-1e28-4a6f-89ce-19250dc5be7a.png)